packages/core/src/hooks/useTokenAllowance.ts

```diff
@@ -0,0 +1,23 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { ERC20Interface } from '../constants'
+import { Falsy } from '../model/types'
+import { useContractCall } from './useContractCall'
+
+export function useTokenAllowance(
+  ownerAddress: string | Falsy,
+  spenderAddress: string | Falsy,
+  tokenAddress: string | Falsy
```

Comment:
`tokenAddress` should be first argument (subject of call first, than parameters)

Reply:
I did it the same way as useTokenBalance, should we change both of them then?

Reply:
Strange. Let's change the order in another PR. I'll add a ticket for that.